eventsourcing/src/main/java/org/axonframework/eventsourcing/configuration/EventSourcingConfigurationDefaults.java

```diff
@@ -57,7 +61,14 @@ public void enhance(@Nonnull ComponentRegistry registry) {
                 .registerIfNotPresent(EventStorageEngine.class,
                                       EventSourcingConfigurationDefaults::defaultEventStorageEngine)
                 .registerIfNotPresent(EventStore.class, EventSourcingConfigurationDefaults::defaultEventStore)
-                .registerIfNotPresent(Snapshotter.class, EventSourcingConfigurationDefaults::defaultSnapshotter);
+                .registerIfNotPresent(Snapshotter.class, EventSourcingConfigurationDefaults::defaultSnapshotter)
+                .registerDecorator(PooledStreamingEventProcessorModule.Customization.class,
```

Comment:
Ah, nice replacement for registering a `StreamableEventSource` bean directly. Good thinking!